### ISSUE TITLE
images_to_video yields unexpected behavior when -r is behind -i 

### ISSUE THREAD

**Issue by yl-1993** [open-mmlab/mmhuman3d#14]
Suppose there are 1202 images under `xxx` folder.

When `-r 30` is behind `-i`, ffmpeg will first expand the total frames to `1202 / 25 * 30 = 1440` and then truncate the first 1202 frames. 
```
ffmpeg -y -threads 4 -start_number 0 -i xxx/frame_%06d.png  -r 30 -frames:v 1202 -c:v libx264 -pix_fmt yuv420p output.mp4
```
Place `-r 30` before `-i` works well
```
ffmpeg -y -threads 4 -start_number 0 -r 30 -i xxx/frame_%06d.png -frames:v 1202 -c:v libx264 -pix_fmt yuv420p output.mp4
```

**Comment by JasonBoy1** [comment]
This will be fixed in PR #12

**Comment by JasonBoy1** [comment]
solved in PR #12
